src/HTML5Backend.js

```diff
@@ -34,25 +34,26 @@ export default class HTML5Backend {
     this.endDragNativeItem = this.endDragNativeItem.bind(this);
   }
 
-  setup() {
-    if (typeof window === 'undefined') {
+  setup(target = window) {
```

Comment:
I have some PRs open that include the ability to use a different window context. After we get done with basic maintenance things I will add this